bin/g5-calc-noise.sh

```diff
@@ -28,73 +27,53 @@ function ReportNoiseAndExit()
 
 if [ -e $INPUT ]; then
   yarr=( $(tail -$MAXRECORDS $INPUT | cut -d ',' -f2 ) )
-  xdate=( $(tail -$MAXRECORDS $INPUT | cut -d ',' -f1 ) )
   n=${#yarr[@]}
 else
-  noise=0
+  noise=0.5
   ReportNoiseAndExit
 fi
 
 #    set initial x values based on date differences
 
 if [ $(bc <<< "$n < $MINRECORDS") -eq 1 ]; then
   # set noise = 0 - unknown
```

Comment:
update comment to "0.5 - light" ?

Reply:
Good point. I will do that and update the comments throughout to reflect the new calculation algorithm